addOns/quickstart/src/main/java/org/zaproxy/zap/extension/quickstart/ExtensionQuickStart.java

```diff
@@ -451,4 +488,27 @@ public void handleNoSavedReport() {
 			}
 		}
 	}
+
+	public void backToMainPanel() {
+		this.getQuickStartPanel().backToMainPanel();
+	}
+
+	private PlugableHud hudProvider;
+
+	public void setHudProvider(PlugableHud hp) {
+		this.hudProvider = hp;

```

Comment:
When null it should disable the HUD button in the panel (otherwise it leads to NPEs).